src/NServiceBus.AcceptanceTests/Performance/When_message_is_audited.cs

```diff
@@ -30,6 +30,26 @@ public class When_message_is_audited : NServiceBusAcceptanceTest
             Assert.IsTrue(context.IsMessageHandledByTheAuditEndpoint);
         }
 
+        [Test]
+        public async Task Should_contain_proper_time_sent()
```

Comment:
@danielmarbach can you be more clear. By meaning together you mean only 1 test? or 2 tests in one class.